packages/addon-dev/sample-rollup.config.js

```diff
@@ -1,15 +1,28 @@
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+
 import { Addon } from '@embroider/addon-dev/rollup';
 
+import packageJson from './package.json';
+
 const addon = new Addon({
   srcDir: 'src',
   destDir: 'dist',
 });
 
+const extensions = ['.js', '.ts'];
+
 export default {
+  // This tells rollup to follow the node-resolution algorithm for finding
+  // dependencies / and files within those dependencies
+  nodeResolve({ extensions }),
+    
   // This provides defaults that work well alongside `publicEntrypoints` below.
   // You can augment this if you need to.
   output: addon.output(),
+  
+  // This prevents peerDependencies from being included in your bundle
+  external: Object.keys(packageJson['peerDependencies']),
```

Comment:
This is great! We need to beef it up though, it should also include:

* The list of Ember's pseudo packages (e.g. `@ember/object` and friends). We should get that list from `@embroider/shared-internals` and expose it on the `Addon` instance we are using here.
* The list of dependencies (the actual resolution / bundling will be done by the final app packager; either ember-auto-import@2 in classic builds or @embroider/webpack in Embroider builds)